vspace/src/main/java/edu/asu/diging/vspace/core/model/SpacesCustomOrder.java

```diff
@@ -0,0 +1,67 @@
+package edu.asu.diging.vspace.core.model;
+
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.OneToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.impl.Space;
+import edu.asu.diging.vspace.core.model.impl.VSpaceElement;
+
+@Entity
+@Inheritance(strategy = InheritanceType.JOINED)
+public class SpacesCustomOrder extends VSpaceElement implements ISpacesCustomOrder {
```

Comment:
Is there a reason you create one object for each space instead of one object that has an order list of spaces? This seems unnecessarily complex and is not very flexible. What if we want to allow different orders depending on certain conditions.